docs/enhancements/decouple-gen-and-run.md

```diff
@@ -0,0 +1,92 @@
+# Customize Sonobuoy Manifest
+
+## Summary
+
+In order to cleanly integrate the sonobuoy client with sonobuoy scanner we need
+a way to customize the objects produced by sonobuoy gen. There are a few
+approaches outlined below ordered by difficulty and benefit.
+
+## Objectives
+
+### Goals
+
+* Allow clients of Sonobuoy to customize the generated manifest and then run it.
+
+### Non Goals
+
+* Make the customizations built in to sonobuoy via config or command line flags.
+
+## Proposal
```

Comment:
Do you have a preferred alternative?

Reply:
Added https://github.com/heptio/sonobuoy/pull/413/files#diff-5644e2d9e24e88115ee8ba37fdca29e6R21